deploy-all

```diff
@@ -1,125 +1,138 @@
+  end
 end
 
 # Vancouver targets
-deploy_standard 'certstrap',       target: 'vancouver', variants: %w(master),              config: 'production'
-deploy_standard 'cf-usb-plugin',   target: 'vancouver', variants: %w(check master)
-deploy_standard 'minibroker',      target: 'vancouver', variants: %w(check master)
-deploy_standard '../fissile-ci',   target: 'vancouver', variants: %w(check master develop)
-deploy_standard '../configgin-ci', target: 'vancouver', variants: %w(check master)
+deploy_vancouver 'certstrap',       variants: %w[master], config: 'production'
+deploy_vancouver 'cf-usb-plugin',   variants: %w[check master]
+deploy_vancouver 'minibroker',      variants: %w[check master]
+deploy_vancouver '../fissile-ci',   variants: %w[check master develop]
+deploy_vancouver '../configgin-ci', variants: %w[check master]
 
 # Nuremberg targets
 Dir.chdir('cf-usb') do
-    status 'deploying cf-usb (openSUSE develop)'
-    run *%w(./deploy -t suse.de production)
-    status 'deploying cf-usb (SLE master)'
-    run *%w(./deploy -t suse.de sle-production-master sle)
+  status 'deploying cf-usb (openSUSE develop)'
+  run './deploy', '-t', 'suse.de', 'production'
+  status 'deploying cf-usb (SLE master)'
+  run './deploy', '-t', 'suse.de', 'sle-production-master', 'sle'
 end
 
 Dir.chdir('../bosh-linux-stemcell-builder-ci') do
-    %w(
-        develop-fissile
-        develop-os-images
-        release-fissile
-        release-os-images
-        release-sles-os-images
-    ).each do |target|
-        status "Deploying bosh-linux-stemcell-builder-ci target #{target}"
-        run *%w(make TARGET=suse.de), target
-    end
+  %w[
+    develop-fissile
+    develop-os-images
+    release-fissile
+    release-os-images
+    release-sles-os-images
+  ].each do |target|
+    status "Deploying bosh-linux-stemcell-builder-ci target #{target}"
+    run 'make', 'TARGET=suse.de', target
+  end
 end
 
 Dir.chdir('../cloudfoundry/ci/pipelines') do
-    begin
-        # This has a bunch of pipelines, but we don't have a good way of
-        # selecting all of the relevant ones
-        run *%w(make pre)
-        %w(
-            pipeline-cf-openstack-validator
-            pipeline-concourse-lftp-resource
-            pipeline-concourse-obs-resource
-            pipeline-helm-charts-check
-            pipeline-helm-charts-sync
-            pipeline-helm-charts-sync-container
-            pipeline-network-test
-            pipeline-opensuse-rootfs-publish
-            pipeline-publish-release-notes
-            pipeline-sle-rootfs-publish
-            pipeline-suse-os-image-stemcell-builder
-            pipeline-suse-packages
-            pipeline-update-handling
-        ).each do |target|
-            status "deploying #{target}"
-            run 'make', 'NON_INTERACTIVE=', target
-        end
-    ensure
-        run *%w(make post)
-    end
+  # This has a bunch of pipelines, but we don't have a good way of
```

Comment:
I don't think it was necessary (rubocop just used the `Dir.chdir` block for that instead).  It's still running `make post` once after the loop.

Reply:
Something is wrong though, because the ensure is still there?

> ./deploy-all:103: syntax error, unexpected keyword_ensure, expecting keyword_end

I've tried on Ruby 2.1.6 and 2.4.2.

Reply:
Ah, this changed in 2.5: https://bugs.ruby-lang.org/issues/12906